bin/ad/ad.c

```diff
@@ -56,7 +56,7 @@ int main(int argc, char **argv)
     if (afp_config_parse(&obj, "ad") != 0)
         return 1;
 
-    setuplog("default:note", "/dev/tty");
+    setuplog("default:note", "/dev/tty", NULL);
```

Comment:
Isn't this a bool parameter and should be "false", not "NULL"?

Reply:
Other then that, lgtm.

Reply:
Thanks for reviewing. I pushed a fix that uses bool "false" for each call of setuplog. Needed to import stdbool.h for a few of the source files.

Reply:
I will squash the commits shortly.